euth/flashpoll/static/js/flashpoll_result.js

```diff
@@ -1,6 +1,12 @@
 /* eslint-disable */
 /* global $ Highcharts */
 
+window.exportResults2 = function (questions, qorderIdIn, key) {
```

Comment:
This looks like debugging code. Also it registers a global variable, please don't do that.